src/index.js

```diff
@@ -20,12 +20,30 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
-ReactDOM.render(
-  <ApolloProvider client={client}>
-    <App />
-  </ApolloProvider>,
-  document.getElementById('root')
-);
+function renderApp(config) {
+  ReactDOM.render(
+    <ApolloProvider client={client}>
+      <App {...config} />
```

Comment:
Does this try (and fail) to iterate over `undefined` when you call the function without arguments?

Reply:
I can't find what JSX attribute spread syntax is supposed to do with `undefined`: https://reactjs.org/docs/jsx-in-depth.html#spread-attributes

Reply:
You're passing it a plain object with no enumerable properties now, this thread's resolved :heavy_check_mark: